Sample manifest json autocomplete in Vs Code and Cursor

prasanta352 opened the pull request:
## Description

Added a new sample demonstrating how to enable JSON schema-based autocomplete and validation for Adobe Express add-on manifest files in VS Code and Cursor editors.

## Related Issue

## Motivation and Context

Developers working on Adobe Express add-ons need better development experience when editing manifest.json files. This sample provides:
  - JSON schema validation for manifest.json files
  - Autocomplete suggestions for valid manifest properties
  - Real-time validation to catch configuration errors early
  - Better IDE integration for add-on development

## How Has This Been Tested?

  - Verified autocomplete works in VS Code with the provided schema configuration
  - Tested with Cursor editor for compatibility
  - Validated that the JSON schema correctly matches Adobe Express add-on manifest structure
## Screenshots (if appropriate):

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

xerxovksi reviewed (commented):
Thanks for putting up this PR! Auto-suggestions for `manifest.json` is a great addition!
I have a few suggestions:
- You can add these .vscode files to the add-on templates [here](https://github.com/adobe/create-ccweb-add-on/tree/main/packages/create-ccweb-add-on/templates). Path: `<template_name>/template/.vscode`
- Another question is how you created the `express-add-on-manifest.schema.json` file - did you create it manually or did you use some tool to generate it? If you are auto-generating, can you add the script [here](https://github.com/adobe/create-ccweb-add-on/blob/main/packages/add-on-manifest/package.json#L35)?

prasanta352 commented:
Thanks for the feedback! 🙌

- I’ve added the `.vscode` files to the add-on templates at `<template_name>/template/.vscode`.
- The `express-add-on-manifest.schema.json` was created manually since the docs don’t fully define all the types (especially optional ones).
- I’ve also added a copy-schema script in the repo to copy the schema files to all the templates.

Opened a PR with these changes here: https://github.com/adobe/create-ccweb-add-on/pull/47

Let me know if there’s anything else I should tweak!
